scripts/opensetup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Install mosquitto
+sudo apt-add-repository ppa:mosquitto-dev/mosquitto-ppa
+sudo apt-get install mosquitto
+sudo apt-get install mosquitto-clients
+ 
+# Generate CA certificate and key
+openssl req -new -x509 -days 365 -extensions v3_ca -keyout my-ca.key -out my-ca.crt 
+
+# Generate private key for MQTT server
+openssl genrsa -out server.key 2048 
```

Comment:
https://unix.stackexchange.com/questions/245778/automate-textual-input-to-a-command-from-a-bash-script